publish/dir.targets

```diff
@@ -89,7 +89,6 @@
                          Exclude="@(_FoundBlobNames)" />
     </ItemGroup>
     <Error Text="Missing publish output from all legs of the build. Missing blobs are '@(_MissingBlobNames)'"
-           Importance="High"
```

Comment:
This is just an existing (unrelated) problem. There is no "Importance" property on Error.